### PR TITLE
Add background blur

### PR DESCRIPTION
**Pull request by javiermolsanz**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -27,36 +27,37 @@ const theme = () => {
...
-    }
+        red: '#D50F2C',
+      },
+    },
   });
 };
 
 function App() {
   const [user, setUser] = useState({
+    backgroundBlur: false,
```
Comment (enricop89) [inline src/App.js:54]: can you change this value to be more extendable in the future? Something like:

```
videoEffects: {
blur: false,
virtualBackground: false
}

```

File: src/components/WaitingRoom/index.js [hunk 2]
```diff
@@ -134,28 +147,61 @@ export default function WaitingRoom({ location }) {
     setLocalVideo(e.target.checked);
   }, []);
 
+  const handleChangeBackgroundBlur = React.useCallback(async () => {
```
Comment (enricop89) [inline src/components/WaitingRoom/index.js:150]: @javiermolsanz looks good. Can you only add a try-catch block?

File: src/hooks/useBackgroundBlur.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      // return track;
+    } catch (e) {
+      console.log('OT get user media error ' + e);
+    }
+  }, []);
+
+  const startBackgroundBlur = async () => {
+    await getUserMedia();
+    backgroundBlur.current = new BackgroundBlurEffect({
+      assetsPath: 'https://d7ca6333nyzk0.cloudfront.net/',
```
Comment (enricop89) [inline src/hooks/useBackgroundBlur.js:25]: weren't you using the ENV Variable?

**Review by enricop89 — COMMENTED** [review]
Added a few comments

File: src/components/WaitingRoom/index.js (hunk 2 above)
Comment (javiermolsanz) [reply src/components/WaitingRoom/index.js:150]: sure thing

**Review by enricop89 — APPROVED** [review]
LGTM - Great Stuff

**Pull request merged**